src/test-apps/ToolCommon.cpp

```diff
@@ -39,7 +39,7 @@
 #include <sys/types.h>
 #include <unistd.h>
 #include <new>
-
+#include <vector>
```

Comment:
Is ToolCommon ever included in on-device (RTOS) tests?

Reply:
No, its not.